dist/debian/debian/rules

```diff
@@ -3,6 +3,7 @@
 include /usr/share/dpkg/pkg-info.mk
 
 export PYBUILD_DISABLE=1
+jobs := $(shell echo $$DEB_BUILD_OPTIONS | sed -r "s/.*parallel=([0-9]+).*/-j\1/")
```

Comment:
How is it preventing the scylla service from starting?

Reply:
@avikivity When trying to install Scylla on Ubuntu:22.04 we get the following error
```
ubuntu@ip-10-0-4-184:~$ sudo systemctl start scylla-server
Job for scylla-server.service failed because the control process exited with error code.
See "systemctl status scylla-server.service" and "journalctl -xeu scylla-server.service" for details.
ubuntu@ip-10-0-4-184:~$ journalctl -xeu scylla-server.service
~
Feb 15 08:16:04 ip-10-0-4-184 systemd[1]: Starting Scylla Server...
░░ Subject: A start job for unit scylla-server.service has begun execution
░░ Defined-By: systemd
░░ Support: http://www.ubuntu.com/support
░░ 
░░ A start job for unit scylla-server.service has begun execution.
░░ 
░░ The job identifier is 1576.
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]: Scylla version 5.3.0~dev-0.20230206.2a5ed115caa0 with build-id f8a20c441f2a20a83ae8d7ce96c5ef8c4203c073 starting ...
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]: command used: "/usr/bin/scylla --log-to-syslog 1 --log-to-stdout 0 --default-log-level info --network-stack posix"
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]: parsed command line options: [log-to-syslog, (positional) 1, log-to-stdout, (positional) 0, default-log-level, (positional) info, network-stack, (positional) posix]
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  seastar - Reactor backend: linux-aio
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  [shard 0] seastar - Creation of perf_event based stall detector failed, falling back to posix timer: std::system_error (error system:13, perf_event_open() failed: Permission denied)
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  [shard 0] seastar - Created fair group io-queue-0, capacity rate 2147483:2147483, limit 12582912, rate 16777216 (factor 1), threshold 2000
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  [shard 0] seastar - IO queue uses 0.75ms latency goal for device 0
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  [shard 0] seastar - Created io group dev(0), length limit 4194304:4194304, rate 2147483647:2147483647
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  [shard 0] seastar - Created io queue dev(0) capacities: 512:2000:2000 1024:3000:3000 2048:5000:5000 4096:9000:9000 8192:17000:17000 16384:33000:33000 32768:65000:65000 65536:129000:129000 131072:257000:257000
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  [shard 1] seastar - Creation of perf_event based stall detector failed, falling back to posix timer: std::system_error (error system:13, perf_event_open() failed: Permission denied)
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  [shard 3] seastar - Creation of perf_event based stall detector failed, falling back to posix timer: std::system_error (error system:13, perf_event_open() failed: Permission denied)
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  [shard 2] seastar - Creation of perf_event based stall detector failed, falling back to posix timer: std::system_error (error system:13, perf_event_open() failed: Permission denied)
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  [shard 0] seastar - updated: blocked-reactor-notify-ms=1000000
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  [shard 1] seastar - updated: blocked-reactor-notify-ms=1000000
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  [shard 2] seastar - updated: blocked-reactor-notify-ms=1000000
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  [shard 3] seastar - updated: blocked-reactor-notify-ms=1000000
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  [shard 0] init - installing SIGHUP handler
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  [shard 0] init - Scylla version 5.3.0~dev-0.20230206.2a5ed115caa0 with build-id f8a20c441f2a20a83ae8d7ce96c5ef8c4203c073 starting ...
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  [shard 0] init - I/O Scheduler is not properly configured! This is a non-supported setup, and performance is expected to be unpredictably bad.
                                             Reason found: none of --max-io-requests, --io-properties and --io-properties-file are set.
                                            To properly configure the I/O Scheduler, run the scylla_io_setup utility shipped with Scylla.
                                            To ignore this, see the developer-mode configuration option.
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  [shard 0] init - Shutting down sighup
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  [shard 0] init - Shutting down sighup was successful
Feb 15 08:16:05 ip-10-0-4-184 scylla[4220]:  [shard 0] init - Startup failed: std::runtime_error (Bad I/O Scheduler configuration)
Feb 15 08:16:05 ip-10-0-4-184 systemd[1]: scylla-server.service: Main process exited, code=exited, status=1/FAILURE
░░ Subject: Unit process exited
░░ Defined-By: systemd
░░ Support: http://www.ubuntu.com/support
░░ 
░░ An ExecStart= process belonging to unit scylla-server.service has exited.
░░ 
░░ The process' exit code is 'exited' and its exit status is 1.
Feb 15 08:16:05 ip-10-0-4-184 systemd[1]: scylla-server.service: Failed with result 'exit-code'.
░░ Subject: Unit failed
░░ Defined-By: systemd
░░ Support: http://www.ubuntu.com/support
░░ 
░░ The unit scylla-server.service has entered the 'failed' state with result 'exit-code'.
Feb 15 08:16:05 ip-10-0-4-184 systemd[1]: Failed to start Scylla Server.
░░ Subject: A start job for unit scylla-server.service has failed
░░ Defined-By: systemd
░░ Support: http://www.ubuntu.com/support
░░ 
░░ A start job for unit scylla-server.service has finished with a failure.
░░ 
░░ The job identifier is 1576 and the job result is failed.
Feb 15 08:16:05 ip-10-0-4-184 systemd[1]: scylla-server.service: Consumed 1.295s CPU time.
░░ Subject: Resources consumed by unit runtime
░░ Defined-By: systemd
░░ Support: http://www.ubuntu.com/support
░░ 
░░ The unit scylla-server.service completed and consumed the indicated resources.
ubuntu@ip-10-0-4-184:~$ 
```

Reply:
Same for debian:11